packages/next/src/server/async-storage/draft-mode-provider.ts

```diff
@@ -11,7 +11,10 @@ import {
 import type { __ApiPreviewProps } from '../api-utils'
 
 export class DraftModeProvider {
-  public readonly isEnabled: boolean
+  /**
+   * @internal - this declaration is stripped via `tsc --stripInternal`
+   */
+  private _isEnabled: boolean
```

Comment:
Prefer different naming than `_` prefix, maybe `enabled`?

Reply:
I followed the existing convention used by the other pseudo-private fields in this class, such as _previewModeId and _mutableCookies.